aio/content/guide/template-syntax.md

```diff
@@ -1342,6 +1342,8 @@ on the asterisk, see the [asterisk (*) prefix](guide/structural-directives#the-a
 
 </div>
 
+When using `*ngIf` on a host element, angular translates this instruction into an `<ng-template>` around the host element, then this template would be controlled by `NgIf` directive.
```

Comment:
Given now `NgIf` is the first place mentioning about **template**, so I'd suggest also lift the explanation of microsyntax expansion to `<ng-template>` to NgIf.